README.md

```diff
@@ -136,6 +136,13 @@ And then building with Cargo as usual:
 $ cargo build
 ```
 
+## Install
+
+After entering the Nix shell as mentioned above, you can install the `lurkrs` binary on your machine with
```

Comment:
The cargo installation method has nothing to do with nix. The first version of the instruction was fine.